randovania/data/json_data/prime3.txt

```diff
@@ -751,48 +834,83 @@ Grand Court Path
 Asset id: 8468936149308094087
 > Dock to Gateway; Heals? False
   * Normal Door to Gateway/Dock to Grand Court Path
+  > Pickup (Missile Expansion)
+      Trivial
```

Comment:
Shouldn't this require PED Suit? I assume it's being shuffled currently but unsure.

Reply:
Too lazy to implement PED suit requirements for now - not really sure how we're handling that

Reply:
Is there something that explicitly requires PED? Or is it just that you can enter hypermode?
If the later, there's an item explicitly for hypermode.

Reply:
Cool, just gonna use that then. Also gonna add it alongside any hyper ball/missile/grapple requirements.

Not entirely sure if we should make hypermode required for rundas/ghor/gandrayda though? I'm leaning towards no

Reply:
Probably not, because its not required on them. The less Hypermode requirements the better imo, so better kept to things that actually need it.